<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1130</width>
    <height>779</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="MainFrame">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1107</width>
      <height>756</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #2b3e47;
border-radius: 25px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="circle1">
     <property name="geometry">
      <rect>
       <x>-90</x>
       <y>330</y>
       <width>351</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle1.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="logo">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>10</y>
       <width>141</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/logoAppWhite.png);</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circle2">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>30</y>
       <width>631</width>
       <height>761</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle1.png);
background-color: transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="GoBack">
      <property name="geometry">
       <rect>
        <x>254</x>
        <y>670</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:transparent;
image:none;
color: #fff;
font: 11pt &quot;Futura Md BT&quot;;
</string>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </widget>
    <widget class="QFrame" name="circle3">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>190</y>
       <width>211</width>
       <height>171</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle2.png);
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="circle4">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>360</y>
       <width>101</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/circle2.png);
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="windowNav">
     <property name="geometry">
      <rect>
       <x>1000</x>
       <y>10</y>
       <width>100</width>
       <height>40</height>
      </rect>
     </property>
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="spacing">
       <number>5</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="minimizeButton">
        <property name="maximumSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../image-classification-and-generation/HomeScreen/home_screen.qrc">
          <normaloff>:/Images/Images/minimize.png</normaloff>:/Images/Images/minimize.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="closeButton">
        <property name="maximumSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../image-classification-and-generation/HomeScreen/home_screen.qrc">
          <normaloff>:/Images/Images/close.png</normaloff>:/Images/Images/close.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>30</width>
          <height>30</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QFrame" name="main_header">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>1111</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QFrame" name="robot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>680</y>
       <width>61</width>
       <height>71</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">image: url(:/Images/Images/robot.png);
background-color:transparent;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>130</y>
       <width>371</width>
       <height>361</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(12, 22, 33);
border-radius : 20px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../image-classification-and-generation/HomeScreen/home_screen.qrc">:/Images/Images/robot.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="generatingLabel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>510</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura Md BT</family>
       <pointsize>10</pointsize>
       <weight>1</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: rgb(255, 255, 255);
border: none;
font: 10 pt &quot;Futura Md BT&quot;;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>550</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QProgressBar{
	background-color:transparent;
	color:transparent;
	text-align:left;
}
QProgressBar::chunk{
	background-color:#f16150;
	border-radius:10px;
}</string>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="loadingStatus">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>577</y>
       <width>201</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura Md BT</family>
       <pointsize>7</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:transparent;
color: rgb(255, 255, 255);
border: none;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="generateButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>437</y>
       <width>141</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura Md BT</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 11pt &quot;Futura Md BT&quot;;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 13pt &quot;Futura Md BT&quot;;
}
</string>
     </property>
     <property name="text">
      <string>Generate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="downloadButton">
     <property name="geometry">
      <rect>
       <x>720</x>
       <y>530</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Futura Md BT</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 #2b3e47, stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 11pt &quot;Futura Md BT&quot;;
}
QPushButton:hover{
	background-color: qlineargradient(spread:pad, x1:0, y1:1, x2:0, y2:0, stop:0 rgb(63, 102, 120), stop:1 #f16150);
	color:#fff;
	border-radius:10px;
	font: 13pt &quot;Futura Md BT&quot;;
}
</string>
     </property>
     <property name="text">
      <string>Download Image</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="generateInput">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>360</y>
       <width>361</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(234, 239, 244, 247);
border : solid 2px rgb(34, 46, 59);
border-radius : 20px;
padding-left : 15px;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string>Express your imagination here !</string>
     </property>
    </widget>
    <widget class="QLabel" name="questionText1">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>240</y>
       <width>111</width>
       <height>51</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(212, 221, 232, 0);
color: rgb(243, 243, 243);
font: 11pt &quot;Futura Md BT&quot;;
text-align : center;</string>
     </property>
     <property name="text">
      <string>Hello Human !</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="questionText1_2">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>270</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgba(212, 221, 232, 0);
color: rgb(243, 243, 243);
font: 11pt &quot;Futura Md BT&quot;;
text-align : center;</string>
     </property>
     <property name="text">
      <string>Tell me what do you want to generate ?</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>logo</zorder>
    <zorder>circle1</zorder>
    <zorder>circle2</zorder>
    <zorder>circle3</zorder>
    <zorder>circle4</zorder>
    <zorder>main_header</zorder>
    <zorder>windowNav</zorder>
    <zorder>robot</zorder>
    <zorder>label</zorder>
    <zorder>generatingLabel</zorder>
    <zorder>progressBar</zorder>
    <zorder>loadingStatus</zorder>
    <zorder>generateButton</zorder>
    <zorder>downloadButton</zorder>
    <zorder>generateInput</zorder>
    <zorder>questionText1</zorder>
    <zorder>questionText1_2</zorder>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="../image-classification-and-generation/HomeScreen/home_screen.qrc"/>
 </resources>
 <connections/>
</ui>
